src/components/markdown/markdown.tsx

```diff
@@ -32,11 +34,15 @@ export class Markdown {
     @Prop()
     public value: string;
 
+    @Prop()
```

Comment:
Since the type is private we need to set this as private as well

Reply:
I made it private (or rather @alpha)